### PR TITLE
<video> tag controls fixes on iOS

### PR DESCRIPTION
**Pull request by sznowicki**
# Pull Request Template

## Description

Workaround for ios <video> tag being covered by Cliplister plugins container (no user interaction possible).

## Type of change

- [x] Bug fix
- Added z-index=1 when video is playing to avoid Cliplister plugins container being on top of the video on iOS.
- Removed deprecated "slot" prop. It's not used in current library and sometimes causes exceptions.
- [ ] New feature
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,10 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+## [1.0.1] - 2018-01-01
```
Comment (Carsten89) [inline CHANGELOG.md]: i guess this date is wrong ;)

File: frontend/helpers/getNavigator.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/**
+ * Navigator as a module, since it's not possible (or difficult) to mock global.navigator
+ * with jest.
+ * @returns {Object}
+ */
+export default () => navigator;
```
Comment (Carsten89) [inline frontend/helpers/getNavigator.js:6]: does it need to be a function?

**Review by Carsten89 — CHANGES_REQUESTED** [review]
one unit test is failing

File: frontend/helpers/getNavigator.js (hunk 2 above)
Comment (sznowicki) [reply frontend/helpers/getNavigator.js:6]: Yes, for unit testing/mocking

**Review by Carsten89 — APPROVED** [review]

**Pull request merged**